loadtest/loadtest.go

```diff
@@ -72,7 +73,13 @@ func (lt *LoadTester) addUser() error {
 	}
 	lt.status.NumUsers++
 	lt.status.NumUsersAdded++
-	controller, err := lt.newController(activeUsers+1, lt.statusChan)
+	userID := activeUsers + 1
```

Comment:
Is this your way of reminding me we should to move from `userId` to `userID` at some point? :grinning:

Reply:
lol .. it's just muscle memory. :)